tornado/ioloop.py

```diff
@@ -911,7 +914,7 @@ def _run(self) -> None:
     def _schedule_next(self) -> None:
         if self._running:
             self._update_next(self.io_loop.time())
-            self._timeout = self.io_loop.add_timeout(self._next_timeout, self._run)
+            self._timeout = self.io_loop.add_timeout(self._next_timeout, self._run)  # type: ignore
```

Comment:
If this works, the type annotation of `add_timeout` should be updated to indicate that it can accept coroutines. so that no `type: ignore` is needed here.